### DISCUSSION TITLE
Seeing file \357\200\274F12\357\200\276 as file

### DISCUSSION THREAD

**Discussion by Rajan-Gupta1** [community/community#63087] [Programming Help]
### Select Topic Area

Question

### Feature Area

Issues

### Body

When I do a git status I get the following. I have never seen this before. What is causing it? Is my git repo infected?
**git status**
On branch main
Your branch is up to date with 'origin/main'.      

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   "\357\200\274F12\357\200\276"

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Hassan247** [comment]
the strange filename your seeing is composed of unicode characters. 
these are some possible causes:
accidentally created by a script or tool mishandling UTF-8 encoding.
someone you're collaborating with added it unknowingly.
less likely, but it could be due to malicious activity.

steps to take:
open the file in a text editor to inspect its content.
if unwanted, unstage with git restore --staged <filename> and delete it.
if concerned about security, run a system malware scan.

**Comment by Spixmacaw** [comment]
the strange filename your seeing is composed of unicode characters.
these are some possible causes:
accidentally created by a script or tool mishandling UTF-8 encoding.
someone you're collaborating with added it unknowingly.
less likely, but it could be due to malicious activity.

steps to take:
open the file in a text editor to inspect its content.
if unwanted, unstage with git restore --staged and delete it.
if concerned about security, run a system malware scan.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Rajan-Gupta1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects.

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
